Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:18.17.1
+
+WORKDIR /app
+
+COPY package*.json .
+
+RUN npm install
```

Comment:
We can use multi-stage building to reduce the count of packages (`npm i --omit=dev` for the 2nd stage), so reduce the size of the image and the count of potential vulnerabilities.